datagateway_api/src/search_api/helpers.py

```diff
@@ -74,8 +84,76 @@ def create_error_message(e):
     return wrapper_error_handling
 
 
+def get_score(entities, query, group, limit):
+    """
+    Gets the score on the given entities based in the query parameter and limit
+    """
+
+    try:
+        myobj = {
+            "query": query,
+            "group": Config.config.search_api.scoring_group,
+            "limit": limit,
+            "itemIds": list(map(lambda entity: entity["pid"], entities)),
+        }
+        response = requests.post(
+            Config.config.search_api.scoring_server, json=myobj, timeout=5
+        )
+
+        if response.status_code < 400:
+            scores = response.json()["scores"]
+            print(str(len(scores)) + " scores retrieved from scoring app")
+            print(str(len(entities)) + " entities retrieved from ICAT")
+            for entity in entities:
+                items = list(
```

Comment:
I'm unsure of how the JSON response will look so I can't immediately comment on this logic. When I test the changes, I'll come back to this comment and give further thoughts